pages/blog/beyond-boundaries.md

```diff
@@ -0,0 +1,83 @@
+
+As an open-source organization, we participate so that we can collaborate with a broader reach of open-source enthusiasts and help bring more folks into the open-source world(which we are already doing). Furthermore, our reason for participating in these programs comes from our passion for open source and the care to support and encourage open-source contributors by building and participating in mentorships.
+
+## Why Start Our Own OSS Mentorship Program?
+
+In 2021, AsyncAPI participated in its first paid mentorship program alongside Postman under Google Summer Of Code(GSoC). After applying to participate as an organization for GSoC in 2021, we got rejected and were opportune to participate alongside Postman.
+
+Postman is an established software development company known for its commitment to open-source initiatives. We received a huge ton of engagement from folks interested in our projects despite having just a few slots allocated to us by Postman.
+
+Fast forward to 2022, we applied for GSoC and we got rejected again this time, Postman also got rejected and this left us in shock and disappointment because a lot of folks were already looking forward to joining us. The shock and disappointment permeated the team's spirits but they refused to let this setback deter them from their mission. 
+
+Determined to overcome this unexpected obstacle, we regrouped and brainstormed alternative strategies to keep the program afloat. What had initially seemed like a devastating rejection transformed into an opportunity for growth and resilience. The team's unwavering passion and refusal to accept defeat turned their setback into a stepping stone toward even greater success.
+
+## The Journey Proved To Be More Challenging Than Anticipated
+
+We knew piloting an in-house mentorship program wasn't going to be easy, that's why we made the initial program a way to collect feedback. We approached the program with a flexible mindset, avoiding rigid decisions or commitments.
+
+During the project ideas phase, [we received 19 project ideas](https://github.com/asyncapi/community/blob/master/mentorship/asyncapi-mentorship/2022/project-ideas.md) and [10 of these projects made it to the final phase](https://github.com/asyncapi/community/blob/master/mentorship/asyncapi-mentorship/2022/README.md) but out of the 10 selected candidates for the program only [six](https://github.com/orgs/asyncapi/discussions/577) were able to complete their project, which isn't bad for our first program, and since we consider a successful program, a program that addresses the real needs of the open-source community and ensuring the program leaders forge meaningful relationships with the participants to ensure their diverse voices are heard and provide them with resources, advice, and networking opportunities.
```

Comment:
```suggestion
During the project ideas phase, [we received 19 project ideas](https://github.com/asyncapi/community/blob/master/mentorship/asyncapi-mentorship/2022/project-ideas.md) and [ten of these projects made it to the final phase](https://github.com/asyncapi/community/blob/master/mentorship/asyncapi-mentorship/2022/README.md) but out of the ten selected candidates for the program only [six](https://github.com/orgs/asyncapi/discussions/577) were able to complete their project, which isn't bad for our first program, and since we consider a successful program. This program addresses the real needs of the open-source community. It ensures the program leaders forge meaningful relationships with the participants to ensure their diverse voices are heard and provide them with resources, advice, and networking opportunities.
```

Reply:
"Program" appears many times. I propose deleting the sentence: "which isn't bad for our first program, and since we consider a successful program."